### PR TITLE
Update for React 16 

### PR DESCRIPTION
**Pull request by shiawuen**
Hi submitting this to make it work with React 16

### CODE REVIEW

**Comment by emilyrohrbough** [comment]
Closes #15

File: package.json [hunk 1]
```diff
@@ -48,8 +48,8 @@
     "eslint-plugin-react": "^6.10.3",
     "jest": "^19.0.2",
     "prettier": "^1.2.2",
-    "react": "^15.5.0",
-    "react-dom": "^15.5.0"
+    "react": "^16.0.0",
+    "react-dom": "^16.0.0"
   },
```
Comment (emilyrohrbough) [inline package.json]: Can we add react and react-dom peerDependencies of `0.14.x || ^15.0.0 || ^16.0.0`?

Comment (shiawuen) [reply package.json]: Done

File: package.json [hunk 2]
```diff
@@ -68,6 +68,9 @@
     ]
   },
   "jest": {
+    "setupFiles": [
+      "<rootDir>/test/__mocks__/requestAnimationFrame.js"
```
Comment (davidtheclark) [inline package.json]: @shiawuen Why was it necessary to add this?

Comment (shiawuen) [reply package.json]: @davidtheclark Because of this https://facebook.github.io/react/blog/#javascript-environment-requirements

Comment (shiawuen) [reply package.json]: React also depends on requestAnimationFrame (even in test environments).

Comment (davidtheclark) [reply package.json]: Aha, thanks!

**Pull request merged**